scripts/computeCLBands.C

```diff
@@ -288,6 +331,52 @@ void computeCLBands(TString configFileName="$GLIKESYS/rcfiles/jointLklDM.rc",Int
     cout << sv2sigmaR[imass] << (imass<nmass-1? "," : "");
   cout << "};" << endl;
 
+  
+  // compute the branching ratios for the branon model for each DM mass
+  if(!channel.CompareTo("branon",TString::kIgnoreCase))
+    {
+      // initialize the array of translation factors for the brane tension limits
+      Double_t braneTensionVal[nmass];
+      for(Int_t imass=shift;imass<nmass;imass++)
+        {
+          Float_t mdm = massval[imass];
+          // release the memory of channelval and brval
+          delete [] channelval;
+          delete [] brval;
+          // number of considering channels
+          nChannels = 9;
+          channelval = new TString[nChannels];
+          brval = new Double_t[nChannels];
+          // call the function that computes the branching ratios and save them in channelval and brval
+          compute_branonBR(mdm,nChannels,channelval,brval,braneTensionVal[imass]);
+        }
+
+      cout << "Double_t bt0sigma_"+channel+"[nmass]  = {";
+      for(Int_t imass=shift;imass<nmass;imass++)
+        cout << 0.001*TMath::Power((braneTensionVal[imass]*1.167*TMath::Power(10., -17.))/sv0sigma[imass], 1./8.) << (imass<nmass-1? "," : "");
```

Comment:
Also add in this section, add the same comment about the origin of the different values.